internal/tx_finder/tx_finder.go

```diff
@@ -75,8 +75,9 @@ func (f Finder) GetMempoolAncestors(ctx context.Context, ids []string) ([]string
 }
 
 func (f Finder) GetRawTxs(ctx context.Context, source validator.FindSourceFlag, ids []string) ([]*sdkTx.Transaction, error) {
+	var err error
 	ctx, span := tracing.StartTracing(ctx, "Finder_GetRawTxs", f.tracingEnabled, f.tracingAttributes...)
-	defer tracing.EndTracing(span)
+	defer tracing.EndTracing(span, err)
```

Comment:
I think we should pass nil here.
1. we do not always stop execution on error
2. it will only log the last error

IMO it would be more valuable if you passed the error to the subtraces `thGetRawTxSpan` and `wocSpan`.